src/Sulu/Bundle/Sales/ShippingBundle/Tests/Functional/Controller/ShippingControllerTest.php

```diff
@@ -372,8 +347,8 @@ private function assertEqualsArrayClass($firstValue, $secondObject, $value)
     }
 
     /**
-     * @param $response
-     * @param $data
+     * @param \stdClass $response
```

Comment:
$response is the response that the ajax request returns. It's type of stdClass since json_decode returns it as that by default.

i.m.o. the naming is correct and the correct type. To avoid confusion i'm going to call it serverResponse